plugin/clang.vim

```diff
@@ -765,9 +791,9 @@ func! s:ClangCompleteInit(force)
   let b:clang_options .= ' -I ' . shellescape(expand("%:p:h"))
 
   " add include directories if is enabled and not ow
+  let l:default_incs = s:DiscoverDefaultIncludeDirs(b:clang_options)
   if g:clang_include_sysheaders && ! l:is_ow
-    let l:incs = s:DiscoverIncludeDirs(g:clang_exec, b:clang_options)
-    for l:dir in l:incs
```

Comment:
Please don't use the old `l:default_incs ` for `g:neocomplete#sources#include#paths[&filetype]`.
Because `b:clang_options ` will be changed when use path in 

```viml
       " Add only absolute paths
       if matchstr(l:dir, '\s*/') != ''
         let b:clang_options .= ' -I ' . shellescape(l:dir)
       endif
```

You should call `s:DiscoverDefaultIncludeDirs(b:clang_options)` again.

In fact, vim-clang is not work well with neocomplete, so it's not important...I think I should remove code related with neocomplete...